ement-room.el

```diff
@@ -988,7 +991,16 @@ spec) without requiring all events to use the same margin width."
                                               (color-darken-name message-color ement-room-prism-message-lightening))))))))
                (redacted-face (when (or local-redacted-by unsigned-redacted-by)
                                 'ement-room-redacted))
-               (body-face (list :inherit (delq nil (list redacted-face context-face type-face)))))
+               ;; For visual consistency, apply the variable-pitch `shr-text' face to
+               ;; non-HTML messages when `ement-room-use-variable-pitch' is non-nil.
+               ;; (HTML messages are fontified by shr itself.)
+               (shr-text-face (when (and ement-room-use-variable-pitch
+                                         (not (equal (or format (alist-get 'format new-content))
```

Comment:
Nitpick: For consistency, would you bind the new-content's format in the pcase-let form?

Reply:
Could do.  I thought it made sense to defer it to here for efficiency, as it saves a bit of unnecessary processing in the cases where it's not being used, which is the vast majority of the cases.

Reply:
Good point, I agree.  Thanks.